File: NumArgs.c
// WORKS
//I am impressed with myself for figuring this out
//I probably should not be, lol
//This program gets the number of arguments passed to it on the command line
//and assigns it to n
//then it prints a string literal and appends the decimal value of N within it
//then it exits
#include <stdio.h>
#include <stdlib.h>
int main (int numargs, char *argv[]) {
   int n;
n = numargs;
   printf ("Number of arguments = %d which is cool\n", n);
   return EXIT_SUCCESS;
} 

File: NumArgs.java
public class NumArgs

//this is a class that prints the number of command line arguments passed to the program when it is run. It shows that Java has a default of 0
//because unlike C it does not need the first argument [0] to "find" itself. 
//This program uses a for loop to iterate through the array of arguments passed to the program, and prints the length of that array. 
{ public static void main(String[] args)
    { 
        
            for(int i=0; i >= args.length; i++) 
            {
                System.out.println("Number of command line args: " + args.length);
                break;
            }
            
            
    }


}